libs/langchain/ragstack_langchain/colbert/colbert_vector_store.py

```diff
@@ -253,22 +255,23 @@ async def afrom_documents(
     def from_texts(
         cls: Type[CVS],
         texts: List[str],
-        embedding: Union[Embeddings, ColbertBaseEmbeddingModel],
+        embedding: Embeddings,
         metadatas: Optional[List[dict]] = None,
         *,
         database: Optional[ColbertBaseDatabase] = None,
         **kwargs: Any,
     ) -> CVS:
-        if isinstance(embedding, Embeddings):
+        if not isinstance(embedding, TokensEmbeddings):
             raise TypeError(
-                "ColbertVectorStore needs a ColbertBaseEmbeddingModel embedding, "
-                "not an Embeddings object."
+                "ColbertVectorStore requires a ColbertEmbeddings embedding."
```

Comment:
change to `"ColbertVectorStore requires a TokensEmbeddings embedding."`?

Reply:
good catch!